lib/read/test/Cardano/Wallet/Read/Tx/TxIdSpec.hs

```diff
@@ -47,24 +37,25 @@ import Test.QuickCheck
     , (===)
     )
 
-import qualified Cardano.Wallet.Read as Read
-import qualified Data.ByteString.Lazy as BL
+import qualified Test.Unit.Cardano.Read.Ledger.Tx as Txs
 
 spec :: Spec
 spec =
     describe "TxId golden tests" $ do
         it "byron tx" $ do
-            prop_matches_TxId byronTx byronTxId
+            prop_matches_TxId Txs.byronTx byronTxId
         it "shelley tx" $
-            prop_matches_TxId shelleyTx shelleyTxId
+            prop_matches_TxId Txs.shelleyTx shelleyTxId
         it "allegra tx" $
-            prop_matches_TxId allegraTx allegraTxId
+            prop_matches_TxId Txs.allegraTx allegraTxId
         it "mary tx" $
-            prop_matches_TxId maryTx maryTxId
+            prop_matches_TxId Txs.maryTx maryTxId
         it "alonzo tx" $
-            prop_matches_TxId alonzoTx alonzoTxId
+            prop_matches_TxId Txs.alonzoTx alonzoTxId
         it "babbage tx" $
-            prop_matches_TxId babbageTx babbageTxId
+            prop_matches_TxId Txs.babbageTx babbageTxId
+        it "conway tx" $
+            prop_matches_TxId Txs.conwayTx conwayTxId
```

Comment:
👍 

In the future, probably worth writing in a way that we can't forget to include an era

Reply:
When the next hardfork comes — this is good enough for now.